torch/quantization/quantize_fx.py

```diff
@@ -0,0 +1,149 @@
+from .fx import Fuser  # noqa: F401
+from .fx import Quantizer  # noqa: F401
+
+def fuse_fx(graph_module, inplace=False):
+    fuser = Fuser()
+    return fuser.fuse(graph_module, inplace)
+
+def _prepare_fx(graph_module, qconfig_dict, inplace, is_dynamic):
+    quantizer = Quantizer()
+    prepare = quantizer.prepare_dynamic if is_dynamic else quantizer.prepare
+    prepared = prepare(graph_module, qconfig_dict, inplace)
+    return prepared
+
+def prepare_fx(graph_module, qconfig_dict, inplace=False):
+    """ If graph_module is in training mode, the model will be
+    prepared as a qat model, otherwise, it will be prepared as a
+    model used for post training static quantization.
+
+    Args:
+      graph_module: model from symbolic_tracing (torch.fx)
+      qconfig_dict: tbd
+    """
```

Comment:
thoughts about adding an assert here that `graph_module` is actually a GraphModule?